README.md

```diff
@@ -1,8 +1,15 @@
+# Upload to SharinPix using URL
```

Comment:
README modified - added backticks and markdown syntax instead of html tags

useful link - https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet